### ISSUE TITLE
Reduce logging or number of stored logs

### ISSUE THREAD

**Issue by 153957** [HiSPARC/datastore#8]
frome now has log level set to DEBUG for the writer and uwsgi processes. With the new PySPARC stations that upload every event individually this causes a lot of extra log messages. We should set the log level to INFO. Or, alternatively, instead of reducing the log level, reduce the number of logs kept by the TimedRotatingFileHandler..

Recently frome was having issues accepting new events due to a full partition, caused by the large logs..

**Comment by 153957** [comment]
The excess of hisparc.log-wsgi logs is caused by [wsgi_app#L107](https://github.com/HiSPARC/datastore/blob/master/wsgi/wsgi_app.py#L107).
This appends the current PID to the log file, which makes it unable to find the old logs to remove.

We could also solve this by running a cron job which removes old files, similar to the [cron job](https://github.com/HiSPARC/publicdb/blob/master/provisioning/publicdb.yml#L110) on the publicdb to remove old .h5 files downloaded using `sapphire.publicdb`

**Comment by 153957** [comment]
Something like:
 `find /var/log/hisparc/ -mtime +100 -delete`

**Comment by tomkooij** [comment]
Rotating logfiles can (should?) be handled by ```syslogd```` or ```logrotate(8)``` or something.

That would be much easier to manage in the future.

**Comment by davidfokkema** [comment]
Yes, well... In the *future*, we should use something more manageable like uWSGI instead of Apache. Then everything can end up in one log, with pid in the log message. uWSGI can also automatically rotate logs, I believe.
